### PR TITLE
CI: Fix packages lint

### PR DESCRIPTION
**Pull request by danigm**
Use the vue-cli-service to lint packages files.

https://phabricator.endlessm.com/T31989

### CODE REVIEW

File: packages/eos-components/src/components/ChannelCardGroup.vue [hunk 1]
```diff
@@ -16,8 +16,8 @@
 
       <!-- eslint-disable vue/no-use-v-if-with-v-for -->
       <b-card
-        v-if="index === rows.length - 1"
         v-for="n in (columns - row.length)"
+        v-if="index === rows.length - 1"
```
Comment (manuq) [inline packages/eos-components/src/components/ChannelCardGroup.vue:20]: I have fixed this and the "node" warning below in my maintenance PR #74, please merge that one first, or I will have to fix conflicts.

File: .pre-commit-config.yaml [hunk 2]
```diff
@@ -19,15 +19,16 @@ repos:
...
-    entry: yarn run lint-frontend
+    entry: yarn run lint-frontend:format
     language: system
     files: ^kolibri_explore_plugin/assets/src/.*\.(js|vue|scss|css)$
   - id: lint-packages
-    name: Linting of JS, Vue, SCSS and CSS files on the packages
+    name: Linting of JS and Vue files on the packages
     description: This hook handles all pacakges linting
-    entry: yarn run lint-frontend
+    entry: yarn run lerna run lint
```
Comment (manuq) [inline .pre-commit-config.yaml:28]: In my maintenance PR I have `yarn lint-packages` that does the same: https://github.com/endlessm/kolibri-explore-plugin/pull/74/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R11

File: .pre-commit-config.yaml [hunk 3]
```diff
@@ -19,15 +19,16 @@ repos:
...
+    pass_filenames: false
     language: system
-    files: ^packages/.*\.(js|vue|scss|css)$
```
Comment (manuq) [inline .pre-commit-config.yaml:30]: Good point. I don't know why but the lint in template-ui or eos-components is not checking errors in the SCSS files.

File: packages/kolibri-api/index.js [hunk 4]
```diff
@@ -1,6 +1,6 @@
 // Communication with Kolibri frontend:
 
-export async function getThumbnail(node) {
+export function getThumbnail(node) {
```
Comment (manuq) [inline packages/kolibri-api/index.js:3]: Why this change? I don't get any errors because of this. And the lint in kolibri-api does nothing, I removed it in my maintenance PR https://github.com/endlessm/kolibri-explore-plugin/pull/74/files#diff-e9e2f9447b84f3e3dd465bcba73a598a60480fd5a788853108b5c5e412b05057L6-L11

**Review by manuq — APPROVED** [review]
Looks good. I have fixed some of this in another PR so I would like that merged first :)

File: packages/kolibri-api/index.js (hunk 4 above)
Comment (danigm) [reply packages/kolibri-api/index.js:3]: I get a lint error because of this rule: https://eslint.org/docs/rules/require-await

Comment (danigm) [reply packages/kolibri-api/index.js:3]: Right now there's no lint in the `kolibri-api` package, so that's the case there's no error, but if you run the command `yarn eslint packages/kolibri-api/index.js` you will see this output:

```
/home/danigm/Projects/work/endless/kolibri/kolibri-explore-plugin/packages/kolibri-api/index.js
  3:8  error  Async function 'getThumbnail' has no 'await' expression  require-await

✖ 1 problem (1 error, 0 warnings)
```

We can add a lint command to this package to catch these issues.

**Comment by danigm** [comment]
Closing, this PR is supersede by #74

File: packages/kolibri-api/index.js (hunk 4 above)
Comment (manuq) [reply packages/kolibri-api/index.js:3]: Good point, I have opened a ticket for tracking this: https://phabricator.endlessm.com/T32022
